src/renderer/components/GroupBase.js

```diff
@@ -128,6 +158,14 @@ class GroupBase extends React.Component {
       </div>
     )
   }
+
+  _getContacts () {
+    const { contacts } = this.props.deltachat
+    if (this.state.showVerifiedContacts) {
+      return contacts.filter(c => c.isVerified === true)
```

Comment:
When creating or editing a verified group, you're only allowed to select between verified contacts.